stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -75,6 +75,18 @@ spec:
 {{ tpl .Values.artifactory.customInitContainers . | indent 6}}
     {{- end }}
       containers:
+      {{- range .Values.artifactory.loggers }}
+      - name: {{ . | replace "_" "-" }}
+        image: busybox
+        command:
+        - tail
+        args:
+        - "-f"
```

Comment:
Better use `-F` so it will wait for logs that were not created yet.

Reply:
Ah I was wondering why my logs stopped when I checked today. Thanks for this 👍